middleware/index.ts

```diff
@@ -151,12 +151,12 @@ function createOrg(logger: Logger, reqContext: any, accessToken?: string): Org {
  * @param functionInvocationId -- FunctionInvocationRequest ID, if applicable
  * @return context
  */
-function createContext(id: string, logger: Logger, reqContext: any, accessToken?: string, functionInvocationId?: string): Context {
-    if (typeof reqContext === 'string') {
+function createContext(id: string, logger: Logger, reqContext?: any, accessToken?: string, functionInvocationId?: string): Context {
+    if (reqContext && typeof reqContext === 'string') {
```

Comment:
should be safe to only check `typeof reqContext` even if `undefined` -- typeof should return `undefined`.